assets/js/xrp-ledger-toml-checker.js

```diff
@@ -254,6 +254,182 @@ function handle_submit(event) {
   fetch_file()
 }
 
+// ------------------------------------------ DOMAIN VERIFICATION VIA ACCOUNT BELOW ------------------------------------------
+let wallet;
+let socket;
+
+
+function makeLogEntry_1(text, raw) {
+  let log
+  if (raw) {
+    log = $('<li></li>').appendTo('#verify-domain-log').append(text)
+  } else {
+    log = $('<li></li>').text(text+" ").appendTo('#verify-domain-log')
+  }
+  log.resolve = function(text) {
+    return $('<span></span>').html(text).appendTo(log)
+  }
+  return log
+}
+
+function fetch_file_1(domain) {
```

Comment:
1. in JS, function name convention is camelCase such as `fetchFile` - original code did not adhere to this standard but we should correct it in this PR 
2. seems like this is the copy of `fetch_file` function - could you refactor the original function to take in a domain param instead of having duplicate code?